src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -276,6 +277,29 @@ export const evidenceSummaryView = ({ formData }) => {
   );
 };
 
+
+/**
+ * @typedef {Object} Disability
+ * @property {String} diagnosticCode
+ * @property {String} diagnosticText
+ * @property {String} ratingPercentage
+ *
+ * @param {Disability} disability
+ */
+export const disabilityOption = ({ disability }) => {
+  const { diagnosticCode, diagnosticText, ratingPercentage } = disability;
+  // May need to throw an error to Sentry if any of these doesn't exist
+
+  return (
+    <div>
+      {diagnosticCode && <h4>{getDiagnosticCodeName(diagnosticCode)}</h4>}
```

Comment:
If we're not sure we'll receive all of this information, should we provide a default generic h4 text for when the info is not provided? (this can be done later if that's preferable)

Reply:
Yeah, that's a great question. I'll write a ticket for it since I think there's a larger discussion to be had surrounding if it even makes sense to let them select it if there isn't this kind of information (since the `/submit` endpoint requires it).